core/templates/services/UpgradedServices.ts

```diff
@@ -69,6 +69,12 @@ import { ConceptCardObjectFactory } from
 import { ContextService } from 'services/context.service';
 import { ContinueValidationService } from
   'interactions/Continue/directives/continue-validation.service';
+import { ContributionOpportunitiesService }
+// eslint-disable-next-line max-len
+  from 'pages/community-dashboard-page/services/contribution-opportunities.service';
```

Comment:
Can we do this:
```
import { ContributionOpportunitiesService }  from
  'pages/community-dashboard-page/services/contribution-opportunities.service';
```
[And remove the pragma]

Reply:
This is resolved

Reply:
@h-ARTS, is this comment resolved?

Reply:
@DubeySandeep This is resolved, PTAL!